src/index.css

```diff
@@ -1,9 +1,229 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@700&display=swap');
+
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
 }
 
 body {
+  margin: 20px;
   text-align: center;
+  align-items: center;
+  margin-left: 12rem;
+  background-color: rgb(204, 209, 211);
+}
+
```

Comment:
- [ ] Great job implementing styles to your project. However, you need to follow the [Bookstore template](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378). One of the requirements is to get the same styling. `(color, background-color, font-size, font-family, and more.)`. 😉 

- [ ] It might be a good idea to add the `ADD NEW BOOK` text in your form.

![bookstore-mucyosoda](https://user-images.githubusercontent.com/79876959/142694174-05e9ee5d-4eb9-48b0-9414-c7d68496bbed.PNG)

![Bookstore-template](https://user-images.githubusercontent.com/79876959/142694189-db8570de-3805-435d-95d9-331f92f44786.PNG)

Reply:
- [ ] Let's complete this requirement from the previous reviewer. The `ADD NEW BOOK` is still not on the page. Also, remember that the more similar to the template the better. I know it can be hard, but you have all of the tools that you need in the Zeplin Template.

You can see in the image below that you can inspect each element to get exact values on colors, size, typographies, spacing, etc.:
![image](https://user-images.githubusercontent.com/79114233/142716657-fde74969-a3a4-4b72-a669-941983b75fac.png)